omnibus/files/server-ctl-cookbooks/infra-server/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -37,10 +37,12 @@
     proxy_send_timeout      300;
     proxy_read_timeout      300;
 
-    <% if !node['packages']['chef-manage'] %>
+    <% unless node['packages']['chef-manage'] %>
     add_header X-Frame-Options DENY;
     add_header X-Content-Type-Options nosniff;
     add_header X-XSS-Protection 1;
+    add_header Permissions-Policy "camera=(); payment=(); microphone=(); gyroscope=(); magnetometer=(); midi=(); geolocation=()";
+    add_header Referrer-Policy 'origin';
```

Comment:
do we want this to be strict-origin instead of origin?

Reply:
I updated to strict-origin to avoid warnings